### PR TITLE
master

### PR DESCRIPTION
**Pull request by dmitry-zhuravlev**
added proxy support in HttpClient

### CODE REVIEW

File: fuel/src/main/kotlin/com/github/kittinunf/fuel/toolbox/HttpClient.kt [hunk 1]
```diff
@@ -78,14 +79,15 @@ class HttpClient : Client {
     }
 
     private fun establishConnection(request: Request): URLConnection {
+        val urlConnection = if (proxy != null) request.url.openConnection(proxy) else request.url.openConnection(proxy)
```
Comment (dmitry-zhuravlev) [inline fuel/src/main/kotlin/com/github/kittinunf/fuel/toolbox/HttpClient.kt]: Sorry. Added some correction.

**Comment by kittinunf** [comment]
I have no problem merging this. Can you please rebase against `master`?

**Comment by dmitry-zhuravlev** [comment]
Done

**Comment by kittinunf** [comment]
Thanks

**Pull request merged**